ondewo/nlu/context.proto

```diff
@@ -108,6 +107,23 @@ service Contexts {
 
 // Represents a context.
 message Context {
+
```

Comment:
i spent ages trying to figure out what was wrong with my mini grpc client the other day till filip told me about this :'(

such is the risk I take always using develop I suppose

Syntax does seem clearer, and you are also getting rid of a google dependency?

Reply:
You mean the Struct dependency?
Yes, that won't be necessary anymore. Not in this file :)